forge-gui/res/cardsfolder/upcoming/showstopping_surprise.txt

```diff
@@ -0,0 +1,8 @@
+Name:Showstopping Surprise
+ManaCost:3 R R
+Types:Instant
+A:SP$ Pump | Cost$ 3 R R | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | RememberObjects$ Targeted | SubAbility$ DBTurnFaceUp | StackDescription$ None | SpellDescription$ Choose target creature you control. Turn it face up if it’s face down. Then it deals damage equal to its power to each other creature.
```

Comment:
what's the point of the extra Pump?

Reply:
This is the same pump method used in the script for Chandra's Ignition. There may be a more concise way to do this, but thought it might be a more secure way given the creature must also be turned face-up if face down before damage is dealt.

Reply:
_Chandra's Ignition_ is a bit different, with the Pump acting as target helper (and it still doesn't need remembered)
Here you can just target with the SetState effect

Reply:
Okay, I am trying it with the opening SetState effect and without any Remembered$ functionality. I'm struggling with the right `Validcards$` tag for the damage part. There appears to be no such parameter that means "all creature this spell doesn't target." I've seen `Creature.IsNotRemembered' for Chandra's Ignition, but I'm looking for something like `Creature.IsNotTargeted` or `Creature.nonTargeted` here. I couldn't find any scripts with a searchable implementation of this. Any thoughts?

Reply:
the script you mentioned already uses `NotDefinedTargeted`

Reply:
I figured out why that use never appeared for me -- an old .rar card script repository used a different script for Ignition. Your suggested way works just fine in the newer snapshot. Thank you for your help and patience!